frontend/src/components/profile/EditableProfileContent.tsx

```diff
@@ -0,0 +1,377 @@
+        onChange={(e) => setGender(e.currentTarget.value as GenderKey)}
+      >
+        <option aria-label="None" value="" />
+        {Object.keys(GENDER_CHOICES).map((key) => (
+          <option key={key} value={key}>
+            {GENDER_CHOICES[key]}
+          </option>
+        ))}
+      </Select>
+    </FormControl>,
+
+    // INTERESTS
+    <Autocomplete
+      key={"editableContent-interests"}
+      multiple
+      clearOnBlur
+      className={classes.textField}
+      options={interestOptions}
+      groupBy={(interest) => interest.category}
+      getOptionLabel={(interest) => interest.title}
+      getOptionSelected={isInterestEqual}
+      value={editableContent.interests}
+      onChange={(e, interests: Interest[]) => setInterests(interests)}
+      inputValue={interestInputValue}
+      onInputChange={(e, value: string) => setInterestInputValue(value)}
+      renderOption={(interest) => `${interest.category}: ${interest.title}`}
+      renderTags={(interest: Interest[], getTagProps) =>
+        interest.map(({ category, title }: Interest, index: number) => (
+          <Chip
+            key={`interestChip-${index}`}
+            variant="outlined"
+            label={`${category}: ${title}`}
+            {...getTagProps({ index })}
+          />
+        ))
+      }
+      renderInput={(params) => (
+        <TextField {...params} variant="outlined" label="Interests" />
+      )}
+    />,
+  ]
+
+  const handleSaveContent = () => {
+    const updates = getProfileContentUpdates(readonlyContent, editableContent)
+    request({
+      path: "/api/profile/",
+      method: "put",
+      body: updates,
+    })
+      .then(() => {
+        setProfileContent(editableContent)
+        toggleEditMode(false)
+      })
+      .catch((error) => {
+        console.log(error)
+        setErrorMessage(error)
+      })
+  }
+
+  const handleEditContent = () => {
+    toggleEditMode(true)
+  }
+
+  const handleCancelEdit = () => {
+    setErrorMessage("")
+    toggleEditMode(false)
+
+    // Reset editable content values.
+    setFirstName(readonlyContent.first_name)
+    setLastName(readonlyContent.last_name)
+    setBirthDate(readonlyContent.birth_date)
+    setGender(readonlyContent.gender)
+    setInterests(readonlyContent.interests)
+  }
+
+  return (
+    <div>
+      {isEditMode ? (
+        <Grid container justify="center" spacing={2}>
+          <Grid item xs={12}>
+            <Grid
+              container
+              alignItems="center"
+              justify="space-evenly"
+              spacing={2}
+            >
+              <Grid item>
+                <Avatar src={""} className={classes.profileAvatar} />
+              </Grid>
+              <Grid item>
+                {editableContent.video ? (
+                  <video
+                    controls
+                    src={editableContent.video}
+                    className={classes.introVideo}
```

Comment:
Instead of the raw video component, can you add a `className` to the `components/video/Video` component and use that?

Reply:
sounds good, the RecordPage also uses a raw video should we change it there as well?